### PR TITLE
Astrology new wound message catch

### PR DESCRIPTION
**Pull request by Dartellum**
Also made def center_telescope(target) formatting to match def peer_telescope.

### CODE REVIEW

**Comment by Dartellum** [comment]
Prior to update:
```
2020-11-28 12:12:43 -0500:[astrology]>center telescope on Szeldia
2020-11-28 12:12:44 -0500:You put your eye to the mahogany telescope.
2020-11-28 12:12:44 -0500:Your vision is too fuzzy, and you are forced to stop.
2020-11-28 12:12:44 -0500:Roundtime: 5 seconds.
2020-11-28 12:12:44 -0500:You feel fully rested.
2020-11-28 12:12:45 -0500:[DRPrime]-DR:Briarleigh: "your prayers to RNGeezus have not been answered"
2020-11-28 12:12:48 -0500:[astrology]>peer telescope
2020-11-28 12:12:48 -0500:...wait 1 seconds.
2020-11-28 12:12:49 -0500:[astrology]>peer telescope
2020-11-28 12:12:49 -0500:You put your eye to the mahogany telescope.
2020-11-28 12:12:49 -0500:Your vision is too fuzzy, and you are forced to stop.
2020-11-28 12:12:49 -0500:Roundtime: 5 seconds.
2020-11-28 12:12:53 -0500:[DRPrime]-DR:Mitkiahn: "I was stanced shield. That'll do it"
2020-11-28 12:12:54 -0500:[astrology]>center telescope on Szeldia
2020-11-28 12:12:54 -0500:You put your eye to the mahogany telescope.
2020-11-28 12:12:54 -0500:Your vision is too fuzzy, and you are forced to stop.
2020-11-28 12:12:54 -0500:Roundtime: 5 seconds.
2020-11-28 12:12:56 -0500:[DRPrime]-DR:Mitkiahn: "Disregard."
2020-11-28 12:12:56 -0500:You feel fully attuned to the mana streams again.
2020-11-28 12:12:59 -0500:[astrology]>peer telescope
2020-11-28 12:12:59 -0500:You put your eye to the mahogany telescope.
2020-11-28 12:12:59 -0500:Your vision is too fuzzy, and you are forced to stop.
2020-11-28 12:12:59 -0500:Roundtime: 5 seconds.
2020-11-28 12:13:01 -0500:[DRPrime]-DR:Damiza: ":P"
2020-11-28 12:13:04 -0500:[astrology]>center telescope on Szeldia
2020-11-28 12:13:05 -0500:You put your eye to the mahogany telescope.
2020-11-28 12:13:05 -0500:Your vision is too fuzzy, and you are forced to stop.
2020-11-28 12:13:05 -0500:Roundtime: 5 seconds.
```
This went on and on and on until I killed astrology.

After the update:
```
2020-11-28 12:21:43 -0500:[astrology]>center telescope on Szeldia
2020-11-28 12:21:44 -0500:You put your eye to the mahogany telescope.
2020-11-28 12:21:44 -0500:Your vision is too fuzzy, and you are forced to stop.
2020-11-28 12:21:44 -0500:Roundtime: 5 seconds.
2020-11-28 12:21:48 -0500:[astrology]>peer telescope
2020-11-28 12:21:48 -0500:...wait 1 seconds.
2020-11-28 12:21:49 -0500:[astrology]>peer telescope
2020-11-28 12:21:49 -0500:You put your eye to the mahogany telescope.
2020-11-28 12:21:49 -0500:Your vision is too fuzzy, and you are forced to stop.
2020-11-28 12:21:49 -0500:Roundtime: 5 seconds.
2020-11-28 12:21:53 -0500:[astrology]>put telescope in my telescope case
2020-11-28 12:21:53 -0500:...wait 1 seconds.
2020-11-28 12:21:54 -0500:[astrology]>put telescope in my telescope case
2020-11-28 12:21:54 -0500:You put your telescope in the telescope case which is inside your oilcloth rucksack.
2020-11-28 12:21:54 -0500:--- Lich: safe-room active.
```

File: common-moonmage.lic [hunk 1]
```diff
@@ -49,12 +49,16 @@ module DRCMM
...
              'Roundtime')
   end
 
   def center_telescope(target)
-    DRC.bput("center telescope on #{target}", 'Center what', 'You put your eye', 'open it to make any use of it', 'The pain is too much', "That's a bit tough to do when you can't see the sky", 'Your search for')
+    DRC.bput("center telescope on #{target}", 'Center what',\
+             'You put your eye', 'open it to make any use of it',\
+             'The pain is too much',\
+             "That's a bit tough to do when you can't see the sky",\
+             'Your search for', 'Your vision is too fuzzy')
```
Comment (KatoakDR) [inline common-moonmage.lic:61]: ❓ Are the slashes `\` necessary at the end of these lines? I'm not a ruby expert and I thought the slahes were only necessary for creating a single string but splitting up the concatenations across multiple lines. I didn't think it'd be necessary if you're formatting the arguments in a method call. 

Ok, did some more googling before posting my question. Per Ruby style guide, I think the `\` are only for [string concatenation](https://rubystyle.guide/#no-trailing-backslash). Method arguments [can be on multiple lines](https://rubystyle.guide/#no-double-indent).

**Comment by KatoakDR** [comment]
Other than my [formatting question](https://github.com/rpherbig/dr-scripts/pull/4629#discussion_r532098516), looks good to me, thanks 👍

File: common-moonmage.lic (hunk 1 above)
Comment (Dartellum) [reply common-moonmage.lic:61]: I just followed what was already there for def peer_telescope. I could remove the slash and see if it still works.

**Pull request merged**